sections/futures/MobileTrade/drawers/TradePanelDrawer.tsx

```diff
@@ -15,7 +16,11 @@ const TradePanelDrawer: FC<TradePanelDrawerProps> = ({ open, closeDrawer }) => {
 			<Background>
 				<Closer onClick={closeDrawer} />
 				<Foreground>
-					<TradeIsolatedMargin mobile />
+					{process.env.NEXT_PUBLIC_CLOSE_ONLY === 'true' ? (
+						<CloseOnlyPrompt mobile />
+					) : (
+						<TradeIsolatedMargin mobile />
```

Comment:
I think instead of having this duplicated in two places we could move it just render the close prompt inside the `TradePanel` instead.

Reply:
Good idea. Updated.